apollo-router/src/plugins/telemetry/config_new/selectors.rs

```diff
@@ -1357,6 +1357,39 @@ impl Selector for SubgraphSelector {
                     .canonical_reason()
                     .map(|reason| reason.into()),
             },
+            SubgraphSelector::SubgraphOperationKind { .. } => response
```

Comment:
I'm pretty sure that this file has unit tests for all these selectors, so we should add to them passing in the response event.